src/ml/neural_net/style_transfer/mps_style_transfer_decoding_node.h

```diff
@@ -25,6 +25,7 @@ API_AVAILABLE(macos(10.14))
                          descriptor:(TCMPSDecodingDescriptor *)descriptor
                         initWeights:(NSDictionary<NSString *, NSData *> *) weights;
 
+- (void) updateIndex:(NSUInteger)index;
```

Comment:
"index" is extremely generic. Perhaps "styleIndex"? Why not just `setStyleIndex`?

Reply:
Fixed!